### ISSUE TITLE
Key-point related queries.

### ISSUE THREAD

**Issue by QDemo** [Microsoft/human-pose-estimation.pytorch#85]
Hi,
I am trying to perform inference by using the pre-trained model (resnet150_coco). Need help on the following:
1. I am not able to map key-points to the original body part labels in the coco dataset format. Can you tell me the output format that is produced from the model (for the keypoints) so that I can Map keypoints at inference to their corresponding body part label.
2. Do we need to compulsorily fix bounding boxes for each person to get the keypoints during inference.
3. For multi-person keypoint generation, do we have to infer the frame multiple times to get the key-points or can we get them only in one inference of the image. If one inference is required then what is the data format the model outputs to differentiate people and their key-points. 

Any help is greatly appreciated.
